exec-tests

```diff
@@ -228,6 +228,11 @@ if [[ "${major}" == "all" || "${major}" == "server" ]]; then
         ${_ECHO} verify_make_source_tree server || rc=1
     fi
 
+    # Verify no database migrations would be generated.
+    if [[ "${subtst:-alembic}" == "alembic" && ${rc} -eq 0 ]]; then
+        (cd lib/pbench/server/database && ${ECHO} alembic check) || rc=1
+    fi
```

Comment:
At line 233, I assume that `ECHO` should start with an underscore (like it does at line 228).

Reply:
This should have been fixed by its removal entirely.